README.md

```diff
@@ -39,13 +39,13 @@ Commit your code regularly and use descriptive messages. This helps both you (in
 Demonstrate your understanding of this week's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
 - [ ] What is the purpose of using _sessions_?
-
+        sessions are used to hide the actual values from the client and you also control when the data expires. This help to prevent hackers.
 - [ ] What does bcrypt do to help us store passwords in a secure manner.
-
+        bcrypt helps to hash the password so it is not in plain text. It takes tha password from the user and the hash the amount of time the programmer set it to. the higher the number the longer the hashing process takes. Once the password is hash it is then shown in the hash format instead of the plain text
```

Comment:
With a JWT we encode the token.  What's the difference between password hashing and jwt encoding?